### PR TITLE
HW CSS 2nd

### PR DESCRIPTION
**Pull request by kissipat**
Hi check please my hw
made all except 'flex'. I read about commands and rules to sort blocks but cant understand how to make mine goal.

I should make one flex containter and write rules or i should make many flex containters (one for navigation block, one for 2 stories, and one for lower block) and rules each other?

### CODE REVIEW

**Comment by pluhin** [comment]
> I should make one flex containter and write rules or i should make many flex containters (one for navigation block, one for 2 stories, and one for lower block) and rules each other?

In this tasks should be many, but not each other

File: aprotos/07_CSS2/hw_5flex.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      .containter {
+      display: flex;
+
+      }
+      .navigation{
+      background-color:grey;
+      }
+      .content{   
+      background-color: purple; 
+
```
Comment (pluhin) [inline aprotos/07_CSS2/hw_5flex.html]: missing }

File: aprotos/07_CSS2/hw_5flex.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
```
Comment (pluhin) [inline aprotos/07_CSS2/hw_5flex.html:1]: This task is not finished

**Review by pluhin — CHANGES_REQUESTED** [review]

File: aprotos/07_CSS2/hw_5flex.html (hunk 2 above)
Comment (kissipat) [reply aprotos/07_CSS2/hw_5flex.html:1]: now finished
used other student's work as a exmaple

File: aprotos/07_CSS2/hw_5flex.html [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+         margin-bottom: .5em;
+         background-color: green;
+         display: flex;
+         flex-direction: column;
+         flex: 0 0 80%;
+         
+      }
+      .lowwer {
+         background-color: grey;
+         display: flex;
```
Comment (pluhin) [inline aprotos/07_CSS2/hw_5flex.html]: Not needed here (flex)

File: aprotos/07_CSS2/hw_5flex.html [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+         flex: 1 1 100%;
+      }
+      .send {
+         display: flex;
```
Comment (pluhin) [inline aprotos/07_CSS2/hw_5flex.html]: Not needed here (flex), use float

**Review by pluhin — CHANGES_REQUESTED** [review]

File: aprotos/07_CSS2/hw_5flex.html (hunk 4 above)
Comment (kissipat) [reply aprotos/07_CSS2/hw_5flex.html]: why not flex? if i delete and use float my pictrure make size = all width

Comment (pluhin) [reply aprotos/07_CSS2/hw_5flex.html]: for picture need just float

**Review by pluhin — CHANGES_REQUESTED** [review]

**Comment by kissipat** [comment]
its ok now?

**Comment by pluhin** [comment]
> I should make one flex containter and write rules or i should make many flex containters (one for navigation block, one for 2 stories, and one for lower block) and rules each other?

display: flex - just for parents, whose children need to organize

File: aprotos/07_CSS2/hw_5flex.html [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+            <li>Каталог</li>
+            <li>Каталог</li>
+            <li>О нас</li>
+            <li>Контакты</li>
+            <li>Информация</li>
+            <li>Подробнее</li>
+            <li>Обратная связь</li>
+      </ul>
+      </div>
+         <div>
```
Comment (pluhin) [inline aprotos/07_CSS2/hw_5flex.html]: here should be class which provide flex: ? ? ?, you need to put a rule for this "column"

File: aprotos/07_CSS2/hw2.html [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+<html>
+    <head>
+<meta charset="utf-8">
+   <title>Code</title>
+   <style>
+   .cont{
+      background-color:black;
+      color:green;
+      width: 400px;
+      max-height: 4em;
```
Comment (pluhin) [inline aprotos/07_CSS2/hw2.html]: better rem

File: aprotos/08_CSS3/css081.html [hunk 7]
```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
```
Comment (pluhin) [inline aprotos/08_CSS3/css081.html:1]: where is media here? task is not finished

**Review by pluhin — CHANGES_REQUESTED** [review]

File: aprotos/08_CSS3/css081.html (hunk 7 above)
Comment (kissipat) [reply aprotos/08_CSS3/css081.html:1]: media in css file

**Comment by kissipat** [comment]
its ok now?

**Comment by kissipat** [comment]
here i tried to make additional task, please, check this also

**Review by pluhin — APPROVED** [review]

**Pull request merged**